recipes/ampliconsuite/meta.yaml

```diff
@@ -20,28 +20,26 @@ source:
 
 build:
     noarch: python
-    number: 0
+    number: 1
 
 requirements:
     build:
         - python >=3
         - setuptools
-        - unzip
-        - wget
 
     run:
         - python >=3
         - bwa
         - cnvkit
         - Flask
-        - future >=0.18.3
```

Comment:
why was it pinned in the first place? Are you sure it working with older version?